ethereum/core/src/test/java/org/hyperledger/besu/ethereum/mainnet/headervalidationrules/GasLimitElasticityValidationRuleTest.java

```diff
@@ -55,19 +55,19 @@
     return Arrays.asList(
         new Object[][] {
           {20000000, 10000000, 10, true},
-          {20019530, 10000000, 10, true},
```

Comment:
This test was link to this PR https://github.com/ethereum/EIPs/pull/3566
the same result as geth . Did this test changed ?

Reply:
These changes were necessary due to the change from <= to < for the gas limit delta rules.  if these are the same cases geth is using, I will double check the spec to ensure this is the correct behavior at the bounds of the max allowable gasLimit delta.  

Either these cases are right, or the reference test cases are right.

Reply:
If there is this kind of modification  it might be necessary to wrap it on a check is1559Block to not impact the blocks before london IMO but maybe I miss something. And to be sure that Geth has the same result? Or to check  if there is a new version of the test  ? 🤔

Reply:
thanks for catching this.  It was indeed a problem in the reference tests rather than our implementation.